apps/member-profile/app/shared/components/profile.personal.tsx

```diff
@@ -5,6 +5,23 @@ import { Country } from '../core.ui';
 import { CityCombobox, CityComboboxProps } from './city-combobox';
 import { EthnicityMultiCombobox } from './ethnicity-combobox';
 
+export function AllowEmailShareField({
+  defaultValue,
+  error,
+  name,
+}: FieldProps<boolean>) {
+  return (
+    <Form.Field error={error}>
+      <Checkbox
+        defaultChecked={defaultValue}
+        label="Share my email with Chapter Leaders! 🌟 "
```

Comment:
I wasn't sure about the label for this. I was also thinking "Share my email with Chapter Leaders to connect with local ColorStack communities!" but this seems long.

Reply:
What if we did something like this:

```
<Form.Field description="If you go to school where there is a ColorStack chapter, this will allow that chapter leader to reach out to you about local events and opportunities." error={error} label="Would you like to share your email with chapter leaders?">
  <Checkbox
      defaultChecked={defaultValue}
      label="Share my email with chapter leaders! 🌟 "
      name={name}
       value="1"
  />
</Form.Field>
```

It's a bit more verbose, but the checkbox's description itself isn't too long. Thoughts?

Reply:
Yeah, that's great! With this additional description I thought it might be better to move the button `Add Email`, rather than have it in between the description and `Save`. Thoughts on this look? 
<img width="982" alt="Screenshot 2024-04-02 at 12 41 32 AM" src="https://github.com/colorstackorg/oyster/assets/91388965/ac8b2574-be62-43dd-a1e4-32b189e35edf">

Reply:
Think it sounds great. For brevity, perhaps

```
<Form.Field description="If your school has a ColorStack chapter, would you like to stay connected for local events and opportunities?" error={error} label="Would you like to share your email with chapter leaders?">
  <Checkbox
      defaultChecked={defaultValue}
      label="Share my email with chapter leaders! 🌟 "
      name={name}
       value="1"
  />
</Form.Field>
```

Reply:
I was thinking potentially we would want this to allow connections beyond just your school, but all schools in your area (especially useful in NYC for example). So maybe the description could be

_Would you like to share your email with chapter leaders? 
If your local schools have a ColorStack chapter, this will allow that chapter leader to reach out about local events and opportunities. 

Share my email with chapter leaders!_

Reply:
That sounds and looks great so far

Reply:
I think for now, we're only going to share emails with their own school. After we merge this, I can follow-up with Johana if we want to allow all local schools though!

Reply:
Sounds good! I updated the description.